closure-typing/storage.js

```diff
@@ -1,5 +1,28 @@
 function storage() {
+    let assassinsList = {};
 
+    function store(key, value) {
+        if (key && value){
+            assassinsList[key] = value;
+        }
+        if (key && !value){
+            if (key in assassinsList){
+                return assassinsList[key];
+            } else {
+                return "Not found";
+            }
+        }
+
+        return assassinsList;
+    }
+
+    //I believe I am doing something wrong
```

Comment:
Why? You doing everything right, believe in yourself!

P.S. Do not leave comments in code, leave them in pull request if you need them of course